src/uu/du/src/du.rs

```diff
@@ -37,6 +38,27 @@ use winapi::um::winbase::GetFileInformationByHandleEx;
 #[cfg(windows)]
 use winapi::um::winnt::{FILE_ID_128, ULONGLONG};
 
+mod options {
+    pub const NULL: &str = "0";
+    pub const ALL: &str = "all";
+    pub const APPARENT_SIZE: &str = "apparent-size";
+    pub const BLOCK_SIZE: &str = "B";
+    pub const BYTES: &str = "b";
+    pub const TOTAL: &str = "c";
+    pub const MAX_DEPTH: &str = "d";
+    pub const HUMAN_READABLE: &str = "h";
+    pub const K: &str = "k";
+    pub const L: &str = "l";
```

Comment:
please rename  the one letter option by something more explicit
it is not obvious when we read options::L